modules/get-started/pages/broker-admin.adoc

```diff
@@ -17,17 +17,17 @@ To determine whether a command needs the `-X brokers` flag or the `-X admin.host
 * Broker addresses are required for communicating with the Kafka API.
 Provide these addresses by using the `-X brokers` flag for commands related to Kafka broker tasks; for example, xref:reference:rpk/rpk-topic/rpk-topic-create.adoc[`rpk topic create`], xref:reference:rpk/rpk-topic/rpk-topic-produce.adoc[`rpk topic produce`], and xref:reference:rpk/rpk-topic/rpk-topic-consume.adoc[`rpk topic consume`].
 * Admin API addresses are required for communicating with the Admin API.
-Provide these addresses by using the `—-api-urls` flag for commands related to cluster or user tasks; for example, xref:reference:rpk/rpk-cluster/rpk-cluster-health.adoc[`rpk cluster health`], xref:reference:rpk/rpk-cluster/rpk-cluster-maintenance.adoc[`rpk cluster maintenance enable <node_id>`], and xref:reference:rpk/rpk-acl/rpk-acl-user.adoc[`rpk security acl user`]. Note that xref:reference:rpk/rpk-cluster/rpk-cluster-metadata.adoc[`rpk cluster info`] is an exception to this rule, since it requires the `-X brokers` flag.
+Provide these addresses by using the `—-api-urls` flag for commands related to cluster or user tasks; for example, xref:reference:rpk/rpk-cluster/rpk-cluster-health.adoc[`rpk cluster health`], xref:reference:rpk/rpk-cluster/rpk-cluster-maintenance.adoc[`rpk cluster maintenance enable <node_id>`], and xref:reference:rpk/rpk-acl/rpk-acl-user.adoc[`rpk security user`]. Note that xref:reference:rpk/rpk-cluster/rpk-cluster-metadata.adoc[`rpk cluster info`] is an exception to this rule, since it requires the `-X brokers` flag.
```

Comment:
I didn't want to change the reference because I'm unsure how it works or how to test it. But it points to the old page.

Reply:
Checking..